cfme/infrastructure/provider/scvmm.py

```diff
@@ -38,9 +38,14 @@ def _form_mapping(self, create=None, **kwargs):
 
     def deployment_helper(self, deploy_args):
         """ Used in utils.virtual_machines """
+        values = {}
         if 'host_group' not in deploy_args:
-            return {'host_group': self.data.get("host_group", "All Hosts")}
-        return {}
+            values['host_group'] = self.data.get("host_group", "All Hosts")
+        if 'cpu' not in deploy_args:
+            values['cpu'] = self.data.get("cpu", 0)
```

Comment:
If cpu/ram isn't in data, you want 0 as a default instead of 1/1024? Assuming this will fail later in the process if these values are 0.